specs/elements/Column.md

```diff
@@ -4,13 +4,14 @@
 
 | Property | Type | Required | Description | Version |
 | -------- | ---- | -------- | ----------- | ------- |
+| **type** | `"Column"` | Yes | Must be `"Column"`. | 1.0 |
 | **items** | `Element[]` | No | The card elements to render inside the `Column`. | 1.0 |
 | **backgroundImage** | `BackgroundImage`, `uri` | No | Specifies the background image. Acceptable formats are PNG, JPEG, and GIF | 1.2 |
 | **bleed** | `boolean` | No | Determines whether the column should bleed through its parent's padding. | 1.2 |
-| **fallback** | `Column`, `FallbackOption` | No | Describes what to do when an unknown item is encountered or the requires of this or any children can't be met. | 1.2 |
+| **fallback** | `Element`, `FallbackOption` | No | Describes what to do when an unknown element is encountered or the requires of this or any children can't be met. | 1.2 |
 | **minHeight** | `string` | No | Specifies the minimum height of the column in pixels, like `"80px"`. | 1.2 |
-| **separator** | `boolean` | No | When `true`, draw a separating line between this column and the previous column. | 1.0 |
-| **spacing** | `Spacing` | No | Controls the amount of spacing between this column and the preceding column. | 1.0 |
+| **separator** | `boolean` | No | When `true`, draw a separating line at the top of the element. | 1.0 |
```

Comment:
>When `true`, draw a separating line at the top of the element [](start = 35, length = 61)

These changes seem not ideal. I think there's a way to have the description in Column.json override the one coming from Element.

Reply:
Ah I agree, this is actually a pretty poor content regression, let me see what else can be done